runtime/libffi/z/sysvz64.s

```diff
@@ -432,51 +480,360 @@ DARGF    DS 0H                l_DOUBLE in arg area
          LA  14,4             We reached max fprs
          B CONT               Next parameter
 
-*If we have spare gprs, pass up to 12bytes
-*in GPRs. 
-*TODO: Store left over struct to argument area
-*
+*If we have spare gprs, pass up to 24 bytes in GPRs. 
 STRCT    DS 0H
-         L   WRKREG,(2176+(((DSASZ+31)/32)*32)+24)(,4)
-         CFI WRKREG,12
-         BL STRCT2
-         B CONT
-STRCT2   DS 0H
-         LA 15,STRCTS
-         LR WRKREG,GPX
-         SLL WRKREG,2
-         L  15,0(WRKREG,15)
-         BR 15
+         LG  11,0(10,5)
+         LG 15,0(,11)          type->size
```

Comment:
Only branch I could see we have to `STRCT` is through ATABLE - so in the beginning of `ARGLOOP` where we process the args. This load to r15 seems to be unneeded given R15 is overwritten with type-elements. Is this right ? If that is the case we should remove this instruction